### DISCUSSION TITLE
how to change social media preview

### DISCUSSION THREAD

**Discussion by Sha0205** [faahim/happy-birthday#73] [Q&A]
hello @faahim, is there any way that i can change the lydia picture to my own picture and put it as social media preview. pls let me know thanks.

**Comment by houssamtheicon** [comment]
> 

same problem here ;''( but im in a rush

**Comment by moyuand** [comment]
Edit in customize.json 
<img width="1053" alt="image" src="https://github.com/faahim/happy-birthday/assets/57672410/73e2fc71-148d-45a1-a6fd-e164ada53ac9">

**Comment by faahim** [comment]
Hey everyone! 👋 

Sorry, this somehow flew off of my radar. Thanks @moyuand for chiming in. 

I currently don't have the time to jump in and fix it but will come back to it as soon as I can. Meanwhile, here are some pointers:

- The project itself does not set any social preview. It's just the site uses the first images from the page by default. Replacing the default image with your own should update the social preview as well (as @moyuand suggested).
- However, that change in the preview can sometimes take a while to reflect. And I understand that most people come to this project with not much time before the special moment. If you want to force update the social preview, you can add respective social meta tags explicitly. Find more details in [this CSS Tricks article](https://css-tricks.com/essential-meta-tags-social-media/).
- It's not my intention to keep that default image as the default social preview. I'd like it to be removed. I'm going to remove it when I get back on it, but if anyone has time for it, please feel free to put up a PR. Should be a nice and easy one to pick up if you're looking to contribute to open source. :) 

Thanks everyone for checking out the project.
